src/components/createCategory.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+
+
+import axios from 'axios';
+import './styling.css';
+
+import style5 from './styling';
+import style4 from './styling';
+import style from './styling';
+
+
+
+
+class CreateCategory extends Component {
+    constructor(props){
+        super(props);
+        this.state={
+          open: false,
+          category_title:'',
+          category_description:''
+        };
+      } 
+
+      handleClick = (e) =>{
+        e.preventDefault();
+        const CreateCategoryUrl='http://127.0.0.1:5000/create_category';
+        axios.post(CreateCategoryUrl,
+            {
+            category_title: this.state.category_title,
+            category_description: this.state.category_description,   
+        },
+        {headers: {'x-access-token': localStorage.getItem('token')}} 
+    )
+        .then(response => {
+            console.log(response);
+            if(response.status === 201){
+                window.location.reload()
+                this.props.history.push('/yummyrecipes/dashboard');
+            }
+            }).catch(error=> {
+                console.log(error)
```

Comment:
Make sure errors are handled and not simply logged. They can be displayed as notifications